pkg/tracker/uniswapTWAP.go

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) The Tellor Authors.
+// Licensed under the MIT License.
+
+package tracker
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math"
+	"math/big"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+
+	tellorCommon "github.com/tellor-io/telliot/pkg/common"
+	"github.com/tellor-io/telliot/pkg/db"
+	"github.com/tellor-io/telliot/pkg/rpc"
+	uniswapcontract "github.com/tellor-io/telliot/pkg/tracker/uniswap"
+)
+
+// UniswapPriceCumulativeLast will be used to save/load data into/from the DB, so that
+// we can minimize onchain contract calls.
+type UniswapPriceCumulativeLast struct {
```

Comment:
```suggestion
type UniswapPriceCumulative struct {
```
I think this is infered.